### PR TITLE
Allow for server error messages to be shown

### PR DESCRIPTION
**Pull request by dmorgant**
Now that detailed messages are emitted by the REST endpoints in the browser locale (see issue #8317), expose those messages when available to the UI.

### CODE REVIEW

File: dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js [hunk 1]
```diff
@@ -531,8 +545,8 @@ var clientInputDialog = (function() {
             var optionId = 'rb_' + option.value.replace(/\s/g, '__');
             var defaultValue = option.value === fldDefault;
             field +=
-                "       <input id='" + optionId + "' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='" + option.value + "' data-default=" + defaultValue + "></input>" +
                 "       <label for='" + optionId + "' class='tool_modal_radio_button_label'>" +
+                "           <input id='" + optionId + "' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='" + option.value + "' data-default=" + defaultValue + ">" +
```
Comment (lavenal) [inline dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js]: Do we need an </input>?

File: dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js [hunk 2]
```diff
@@ -578,18 +592,16 @@ var clientInputDialog = (function() {
...
             "       <div class='tool_modal_body_field_helper_text'>" + fldDescription + "</div>" +
             "       <div class='tool_modal_radio_button_horizontal_wrapper'>" +
-            "           <input id='" + fldId + "-true' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='true'";
-        field += fldDefault === 'true' ? " data-default='true'" : " data-default='false'";
-        field +=
-            "></input>" +
             "           <label for='" + fldId + "-true' class='tool_modal_radio_button_label'>" +
+            "               <input id='" + fldId + "-true' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='true'";
+        field += fldDefault === 'true' ? " data-default='true'" : " data-default='false'";
+        field += ">" +
```
Comment (lavenal) [inline dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js]: </input>?

File: dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js [hunk 3]
```diff
@@ -578,18 +592,16 @@ var clientInputDialog = (function() {
...
             "               <span class='tool_modal_radio_button_appearance'></span>" + messages.TRUE + 
             "           </label>" +
-            "           <input id='" + fldId + "-false' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='false'";
-        field += fldDefault === 'false' ? " data-default='true'" : " data-default='false'";
-        field += 
-            "></input>" +
             "           <label for='" + fldId + "-false' class='tool_modal_radio_button_label'>" +
+            "               <input id='" + fldId + "-false' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='false'";
+            field += fldDefault === 'false' ? " data-default='true'" : " data-default='false'";
+            field += ">" +
```
Comment (lavenal) [inline dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js]: \</input\>?

File: dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js (hunk 1 above)
Comment (dmorgant) [reply dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js]: No...not in HTML
image

File: dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js (hunk 2 above)
Comment (dmorgant) [reply dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js]: No...not in HTML...from w3schools...

image

File: dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js (hunk 3 above)
Comment (dmorgant) [reply dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js]: No.....Not in HTML...from w3 schools....

image

**Comment by dmorgant** [comment]
Some accessibility improvements were also made in this issue.  Namely, enclosing input fields within the \<label> tags associated with the fields instead of following them.

**Comment by dmorgant** [comment]
Fixes #9999

#build

**Comment by dmorgant** [comment]
#build

**Pull request merged**